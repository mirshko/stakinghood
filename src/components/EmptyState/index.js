import React from "react";
import { Text } from "rebass";

const EmptyState = ({ children }) => (
  <Text
    py={20}
    fontSize={[1, 2]}
    textAlign="center"
    color="var(--brand-opaque)"
  >
    {children}
  </Text>
);

export default EmptyState;
