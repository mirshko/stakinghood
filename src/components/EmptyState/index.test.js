import React from "react";
import { shallow } from "enzyme";

import EmptyState from "./index";

describe("<EmptyState />", () => {
  it("renders the passed in text", () => {
    const wrapper = shallow(<EmptyState>Text</EmptyState>);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.text()).toContain("Text");
  });
});
