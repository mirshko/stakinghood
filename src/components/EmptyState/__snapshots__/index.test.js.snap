// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptyState /> renders the passed in text 1`] = `
<Styled(styled.div)
  color="var(--brand-opaque)"
  fontSize={
    Array [
      1,
      2,
    ]
  }
  py={20}
  textAlign="center"
>
  Text
</Styled(styled.div)>
`;
